README.md

```diff
@@ -246,19 +259,23 @@ is needed, it would be necessary to include it in
 the OpenID Provider's metadata ([RFC 8414](https://tools.ietf.org/html/rfc8414#section-2)) or download it from 
 ``/.well-known/assetlinks.json``.
 
+From Android 11, the [app will need to request extra permissions](https://developer.android.com/preview/privacy/package-visibility) to do this.
+
 
 ### Problem with Android App Links
 If an IDP app is responsible for several hundred domains,
 the app has to register an Android App Link for every single
 domain. If one of these registrations fails, none of them would be valid.
 Additionally, the IDP app has to download a document from every
-single domain. This could be a problem (not only) in mobile networks.
+single domain (JJH: I don't understand this point - did it mean "the OS has to download..."?). This could be a problem in situations like low bandwidth or high latency networks.
 
 To solve this the Relying Party could add the package name to the
 intent that opens the redirect URL. This would ensure that the 
 correct app opens without Android App Links. It is also a good 
 solution since we already have the package name of the IDP
 to check whether the app is installed or not.
+JJH: I don't understand where we already have the package name from
+JJH: I'm worried that this section could be intrepretted as suggesting this very small bit of code is a better/as good solution compared to applinks - which it isn't, because we've lost the signature verification.
```

Comment:
Good point! I will add this and reference the signature verification process in the section "Problem with Alternative App Stores".